tests/wpt/metadata/MANIFEST.json

```diff
@@ -169897,6 +169897,66 @@
      {}
     ]
    ],
+   "_certs/064162.pem": [
```

Comment:
Should these lines really be here? To me they look more like some left-over temporary files.